src/components/chip/examples/chip-progress.tsx

```diff
@@ -0,0 +1,66 @@
+import { Component, h, State } from '@stencil/core';
+
+/**
+ * Displaying a progress bar
+ * By defining a numeric `progress` (between `0` to `100`),
```

Comment:
"between … to …" really is a pet peeve of mine, both in English and in Swedish… 🙈

```suggestion
 * By defining a numeric `progress` (between `0` and `100`),
```

or 

```suggestion
 * By defining a numeric `progress` (from `0` to `100`),
```